lib/model/narrow.dart

```diff
@@ -39,13 +39,9 @@ sealed class SendableNarrow extends Narrow {
   MessageDestination get destination;
 }
 
-/// The narrow called "All messages" in the UI.
-///
-/// This does not literally mean all messages, or even all messages
-/// that the user has access to: in particular it excludes muted streams
-/// and topics.
```

Comment:
I am not sure whether I should remove this comment now or not?

Reply:
I think we can replace it with something like:

```dart
/// All messages the user has access to, excluding unsubscribed streams
/// and muted streams and topics. See [PerAccountStore.isTopicVisible].
```

Reply:
Sounds good done!